packages/atomic/src/components/insight/atomic-insight-search-box/atomic-insight-search-box.tsx

```diff
@@ -385,27 +390,30 @@ export class AtomicInsightSearchBox {
 
   public render() {
     return (
-      <SearchBoxWrapper disabled={this.disableSearch}>
+      <SearchBoxWrapper disabled={this.disableSearch} textArea>
         <atomic-focus-detector
           style={{display: 'contents'}}
           onFocusExit={() => this.clearSuggestions()}
         >
           <atomic-icon
             part="submit-icon"
-            icon={SearchIcon}
-            class="w-4 h-full my-auto mr-0 ml-4"
+            icon={SearchSlimIcon}
+            class="w-4 h-4 my-auto mr-0 ml-4"
           />
-          <SearchInput
-            inputRef={this.inputRef}
+          <SearchTextArea
+            textAreaRef={this.textAreaRef}
             loading={this.searchBoxState.isLoading}
-            ref={(el) => (this.inputRef = el as HTMLInputElement)}
+            ref={(el) => (this.textAreaRef = el as HTMLTextAreaElement)}
```

Comment:
Avoid type casting from `T|undefined` to `T`, instead do eliminate the `undefined` with an assertion (here, `el &&`)

```suggestion
            ref={(el) => el && (this.textAreaRef = el)}
```